material-ui/app/entry.client.tsx

```diff
@@ -0,0 +1,33 @@
+import { RemixBrowser } from "@remix-run/react";
+import { PropsWithChildren, useState } from "react";
```

Comment:
We import types on a separate line

```suggestion
import type { PropsWithChildren } from "react";
import { useState } from "react";
```